ddtrace/settings/asm.py

```diff
@@ -65,7 +64,9 @@ class ASMConfig(Env):
     # prevent empty string
     if _asm_static_rule_file == "":
         _asm_static_rule_file = None
-    _iast_enabled = Env.var(bool, IAST_ENV, default=False)
+    _iast_enabled = Env.var(bool, IAST.ENV, default=False)
+    _iast_debug = Env.var(bool, IAST.ENV_DEBUG, default=False)
```

Comment:
```suggestion
    _iast_debug = Env.var(bool, IAST.ENV_DEBUG, default=False, private=True)
```
the `private=True` automatically adds a `_` in front of the variable name and could be good to exclude these from auto-generated docs

Reply:
I didn't know this param! very useful! thank you @P403n1x87 !